grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -244,6 +246,7 @@ class StockMovementService {
         } else {
             requisition.status = status
             requisition.save(flush: true)
+            publishEvent(new RequisitionStatusTransitionEvent(requisition, requisition.status))
```

Comment:
You could pass the requisition.id here in order to force a read or get on the event service. We could think about doing that across all of our events.

Reply:
Is there any need to pass both the from and to status here?

Reply:
Could you elaborate on what you mean by "from and to" status?